### ISSUE TITLE
关于任务无关的蒸馏

### ISSUE THREAD

**Issue by savannahfan** [airaria/TextBrewer#89]
您好，感觉textbrewer的框架非常清晰，也非常容易使用，但是我看几个example都是与下游任务有关的蒸馏，如果不传入labels的话就是通过attention_mask传入select_logits_with_mask得到预测的logtis，那么就是对所有词进行预测。那么，我可以自己传入mlm labels，进行任务无关的蒸馏吗？谢谢！期待回复

**Comment by airaria** [comment]
MLM其实不也是一种“任务”吗？MLM也有inputs和labels，labels是mask位置的token ids。
MLM形式上和序列标注类似，可以仿照序列标注的做法蒸馏


在2021年12月16日 ***@***.***> 写道：

您好，感觉textbrewer的框架非常清晰，也非常容易使用，但是我看几个example都是与下游任务有关的蒸馏，如果不传入labels的话就是通过attention_mask传入select_logits_with_mask得到预测的logtis，那么就是对所有词进行预测。那么，我可以自己传入mlm labels，进行任务无关的蒸馏吗？谢谢！期待回复

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
